virtualizarr/utils.py

```diff
@@ -13,7 +19,7 @@ def _fsspec_openfile_from_filepath(
     reader_options: Optional[dict] = {
         "storage_options": {"key": "", "secret": "", "anon": True}
     },
```

Comment:
```suggestion
    reader_options: Optional[dict] = {},
```
I belive this should fix it?